### PR TITLE
[NEWDEV][SHELL32][DEVMGR]  Update / Small improvements to pt-PT translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[NEWDEV][SHELL32][DEVMGR]  Update / Small improvements to pt-PT translation

### CODE REVIEW

**Review by binarymaster — APPROVED** [review]

File: dll/win32/devmgr/lang/pt-PT.rc [hunk 1]
```diff
@@ -7,10 +7,10 @@ BEGIN
     IDS_NAME "Nome"
     IDS_TYPE "Tipo"
     IDS_MANUFACTURER "Fabricante: %1"
-    IDS_LOCATION "Local: %1"
-    IDS_STATUS "Estado do dispositivo: %1"
+    IDS_LOCATION "Localização: %1"
+    IDS_STATUS "Estado do\ndispositivo: %1"
```
Comment (binarymaster) [inline dll/win32/devmgr/lang/pt-PT.rc:11]: Are you sure about using "\n" line break here? Other translations does not seem to do that:

image

Comment (binarymaster) [inline dll/win32/devmgr/lang/pt-PT.rc:11]: Are you sure about using "\n" line break here? Other translations does not seem to do that:

image

Comment (zecarlos1957) [reply dll/win32/devmgr/lang/pt-PT.rc:11]: ReactOS
ReactOS2

Improved visual arrangement

Comment (binarymaster) [reply dll/win32/devmgr/lang/pt-PT.rc:11]: The "before" looks better for me. It's also the reason why this label supports word wrapping.

Comment (zecarlos1957) [reply dll/win32/devmgr/lang/pt-PT.rc:11]: OK. I accept the majority opinion. I like the altered version as it is more aligned .

Comment (binarymaster) [reply dll/win32/devmgr/lang/pt-PT.rc:11]: I'm pretty sure this is how it's done in Windows, might be a good idea to double check.

Comment (zecarlos1957) [reply dll/win32/devmgr/lang/pt-PT.rc:11]: Maybe I have a better solution for both.
"Estado disp.." Like in Win7 pt-PT

Comment (binarymaster) [reply dll/win32/devmgr/lang/pt-PT.rc]: Maybe the best option would be to resize the underlying dialog box itself.

Comment (zecarlos1957) [reply dll/win32/devmgr/lang/pt-PT.rc]: I like that solution.

Comment (binarymaster) [reply dll/win32/devmgr/lang/pt-PT.rc]: Hmm... I actually tried to resize the `IDD_HARDWARE` dialog box, and it seems it's not possible, as the property sheet is then resized automatically.

While checking, I looked up the dialog in Restorator, and noticed there's also problem with "fix conflicts" button, which is usually hidden. Here's the fix for it:
```
PUSHBUTTON "&Resolução de problemas...", IDC_TROUBLESHOOT, 42, 140, 97, 14, BS_PUSHBUTTON | WS_CHILD | WS_DISABLED | WS_TABSTOP
```

**Comment by binarymaster** [comment]
Ok let's leave that label as is for now, I also pushed the fix for the button.

**Pull request merged**